test/unit/views/movement/ConsignmentReferenceViewSpec.scala

```diff
@@ -31,7 +31,7 @@ class ConsignmentReferenceViewSpec extends ViewSpec {
 
     "render title" in {
 
-      page(ConsignmentReferences.form()).getTitle must containMessage("consignmentReferences.title")
+      page(ConsignmentReferences.form()).getTitle must containMessage("consignmentReferences.ARRIVE.question")
```

Comment:
I know it's not a part of this PR but I would prefer to have `arrive` not `ARRIVE`

Reply:
Normally I would point out that we have used enum raw values in a few places as part of the message keys.  But as we have time atm I'll change it.   
Just don't expect this level of service all of the time.